tests/DocumentationExamplesTest.php

```diff
@@ -1194,11 +1208,10 @@ public function testRunCommand_example_1(): void
     public function testRunCommand_example_2(): void
     {
         $db = new Database($this->manager, $this->getDatabaseName());
-        $db->dropCollection('restaurants');
-        $db->createCollection('restaurants');
+        $db->createCollection('inventory');
 
         // Start runCommand Example 2
-        $cursor = $db->command(['collStats' => 'restaurants']);
+        $cursor = $db->command(['collStats' => 'inventory']);
```

Comment:
The content between `// Start` and `// End` should not be changed, as this is parsed by the docs team for inclusion in the MongoDB server manual. Examples are expected to be consistent across languages. 

For example, see the language selection widget in the right column on https://www.mongodb.com/docs/manual/tutorial/insert-documents/.

On a somewhat related note: "runCommand Example 2" will probably be removed when we implement [PHPLIB-1065](https://jira.mongodb.org/browse/PHPLIB-1065), but that's outside the scope of this PR.